Xamarin.Forms.Platform.Android/AppCompat/ButtonRenderer.cs

```diff
@@ -170,14 +187,20 @@ void UpdateBitmap()
 
 			var image = Context.Resources.GetDrawable(imageFile);
 
-			if (string.IsNullOrEmpty(Element.Text))
+			if (IsNullOrEmpty(Element.Text))
```

Comment:
It's more concise. Having to class-qualify static methods has always been an awkward part of C#. 
And if anyone is confused they just have to look for the definition of `IsNullOrEmpty` in our code to figure out that we didn't write our own.